_rules/table-headers-attribute-refer-to-data-cells.md

```diff
@@ -0,0 +1,223 @@
+---
+id: a25f45
+name: `headers` attribute only refers to `cells` in the same `table` element
+test_type: atomic
+description: |
+  This rule checks that the `headers` attribute must only refer to `cells` in the same `table` element
+test_aspects:
+- DOM Tree
+authors:
+- Jey Nandakumar
+- Audrey Maniez
+---
+
+## Test Procedure
+
+### Applicability
+
+This rule applies to `td` and `th` elements with a `headers` attribute and within a `table` element with a [semantic role](#semantic-role) of `table` or `grid` or `treegrid`, that is [included in the accessibility tree](#included-in-the-accessibility-tree).
+
+### Expectation
+
+Each target element with the `headers` attribute refers to other `th` of the same `table` element.
```

Comment:
Should we include cells with the columnheader role in addition to `th` elements?

Reply:
If it was a `td` with an id, would referring to that also work? 
I am thinking if it's enough that it's just another cell in the same table...

Reply:
If the `td` that is being referred to, is marked as `row/columnheader` using a `role`.